pop-api/examples/balance-transfer/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "pop_api_extension_demo"
+name = "balances"
```

Comment:
I would call it `balance_transfer`. Same as top-level folder.

What I would do is actually change the top-level folder to be `balances` instead of `balance-transfer`.